runtime/src/prioritization_fee.rs

```diff
@@ -0,0 +1,354 @@
+use {
+    crate::transaction_priority_details::GetTransactionPriorityDetails,
+    solana_measure::measure,
+    solana_sdk::{
+        clock::Slot, pubkey::Pubkey, saturating_add_assign, transaction::SanitizedTransaction,
+    },
+    std::collections::HashMap,
+};
+
+#[derive(Debug, Default)]
+struct PrioritizationFeeMetrics {
+    // Count of writable accounts in slot; including accounts have minimum prioritization fee equal
+    // to blocker's minimum fee, the number of these irrelevant accounts is the difference between
+    // this counter and `relevant_writable_accounts_count`.
+    total_writable_accounts_count: u64,
+
+    // Count of writeable accounts with a minimum prioritization fee higher than the minimum transaction
+    // fee for this slot.
+    relevant_writable_accounts_count: u64,
+
+    // Total prioritization fees included in this slot.
```

Comment:
Improved comment would be good here. Is this total number of transactions that have fees? Total of all fees paid?

Reply:
Not "all fees" nor transaction count, it is "total prioritization fee" for this slot, as the comments indicate 😃